admin-dev/themes/default/template/controllers/images/content.tpl

```diff
@@ -123,10 +123,23 @@
 				</div>
 			</div>
 			<div class="panel-footer">
-				<button type="submit" name="submitRegenerate{$table}" class="btn btn-default pull-right" onclick="return confirm('{l s='Are you sure?' d='Admin.Notifications.Warning'}');">
+        <input type="hidden" name="submitRegenerate{$table}" value="" />
+				<button id="submitRegenerate{$table}" class="btn btn-default pull-right">
```

Comment:
```html
<button 
type="submit" 
name="submitRegenerate{$table}" 
value="" 
id="submitRegenerate{$table}" 
class="btn btn-default pull-right"
>
```

Reply:
Fixed

Reply:
@Progi1984 You can remove the `<input type="hidden" name="submitRegenerate{$table}" value="" />`, it should work the same. 😎

Reply:
Just reverted your fix. Because the modal doesn't POST the button data, so it was the cause I implemented an hidden input.